R/Sound_DMR.R

```diff
@@ -748,30 +745,29 @@ individual_DMR <- function(Output_Frame, ZoomFrame_filtered, experimental_design
 #' @return Output_Frame (df)
 #' @export
 
-find_DMR <- function(Output_Frame, dmr_obj, fixed = c('Group'),
-                     random = c('Plant'), reads_threshold = 3,
-                     model, control = '', analysis_type) {
+find_DMR<- function(methyl_summary, dmr_obj, fixed = c('Group'),
```

Comment:
For this function name, do we want it to be the same as above, "methylsummary"? Also in the workflow we have a "methyl_summary" variable, didn't we have some concern with using the same variable name in these function environments that we use in the workflow environment?

Reply:
Apparently issue was using the Output_frame variable within nested functions for both input and output. This is probably not gonna cause issues since we ran tests but i'd be up for changing this @jcolicchio-soundag just to avoid any confusions in the future.